.github/pull_request_template.md

```diff
@@ -0,0 +1,62 @@
+[//]: # (This is a template of a pull request template.)
+[//]: # (You should modify it considering specifics of a particular repository and put it there.)
+[//]: # (Comments like this are meta-comments, they shouldn't be present in the final template.)
+[//]: # (Comments starting with '<!---' are intended to stay in the final template.)
+
+[//]: # (Keep in mind that it's only a template which contains items relevant to almost all conceivable repository.)
+[//]: # (There can be other important items relevant to your repository that you can add here.)
+
+## Description
+
+<!--
+Describes the nature of your changes. If they are substantial, you should
+further subdivide this into a section describing the problem you are solving and
+another describing your solution.
+-->
+
+[//]: # (Here you can add a link to the corresponding issue tracker, e. g. https://issues.serokell.io/issue/AD-)
+[//]: # (For GitHub/GitLab issues it is better to use just hash symbol or exclamation mark as it is more resistant to repo movements)
+[//]: # (In this case please also prefix the link with "Resolves" keyword)
+[//]: # (See https://docs.gitlab.com/ee/user/project/issues/managing_issues.html#closing-issues-automatically)
+[//]: # (See https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords)
+## Related issue(s)
+
+<!--
+- Short description of how the PR relates to the issue, including an issue link.
+For example
+- Fixed #100500 by adding lenses to exported items
+
+Write 'None' if there are no related issues (which is discouraged).
+-->
+
```

Comment:
I suggest adding a hint:
```suggestion

Resolves TCFMM-

```
(IMO the issue number is enough)

Reply:
Updated.